foundry.toml

```diff
@@ -21,6 +21,9 @@ remappings = [
   '@solady=lib/optimism/packages/contracts-bedrock/lib/solady/src',
   'ds-test/=lib/optimism/packages/contracts-bedrock/lib/forge-std/lib/ds-test/src',
   'forge-std/=lib/forge-std/src/',
+  'safe-contracts/=lib/optimism/packages/contracts-bedrock/lib/safe-contracts/contracts/',
+  'safe-contracts-v1.3.0/=lib/safe-contracts/contracts/',
```

Comment:
Can you remind me why we have this twice? I believe all safe's are on 1.3.0, but the monorepo version is different?

Reply:
Yes, that's correct. Mainly this is so that we can use the `GnosisSafe.sol` naming here of v1.3.0, rather than `Safe.sol` naming of v1.4.1 as used in the monorepo. 

I don't recall all of the details, but I have spent a fair bit of time in the past trying to avoid having to do this, and quite a few challenges crop up. I believe they are mostly related to the dependency graph, remappings and the import names used across this repo, `base-contracts` and the monorepo.